configure.ac

```diff
@@ -828,6 +848,19 @@ AM_CONDITIONAL([CVC4_USE_CRYPTOMINISAT], [test $have_libcryptominisat -eq 1])
 AC_SUBST([CRYPTOMINISAT_LDFLAGS])
 AC_SUBST([CRYPTOMINISAT_LIBS])
 
+# Build with LFSC
+AC_ARG_WITH([lfsc],
+  [AS_HELP_STRING([--with-lfsc],
+     [use the LFSC proof checker])], [], [with_lfsc=])
+CVC4_CHECK_FOR_LFSC
+if test $have_liblfsc -eq 1; then
+  CVC4CPPFLAGS="${CVC4CPPFLAGS:+$CVC4CPPFLAGS }-DCVC4_USE_LFSC"
```

Comment:
Hm, this is analogous to how it is done for abc and cryptominisat and since we're gonna switch to cmake anyways I'd rather keep it as it is done for the others.